src/rail/pipelines/degradation/truth_to_observed.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Prerquisites, os, and numpy
+import os
+import numpy as np
+
+# Various rail modules
+from rail.tools.photometry_tools import Dereddener, Reddener
+
+from rail.core.stage import RailStage, RailPipeline
+
+import ceci
+
+from rail.core.utils import RAILDIR
+
+if 'PZ_DUSTMAP_DIR' not in os.environ:
+    os.environ['PZ_DUSTMAP_DIR'] = '.'
+
+dustmap_dir = os.path.expandvars("${PZ_DUSTMAP_DIR}")
+
+from .spectroscopic_selection_pipeline import SELECTORS, CommonConfigParams
+from .apply_phot_errors import ERROR_MODELS
+
+
+class TruthToObservedPipeline(RailPipeline):
+
+    default_input_dict = dict(input='dummy.in')
```

Comment:
Yes.  It gets used in rail.core.stage.RailPipeline.build_and_write at line 128 so that ceci thinks that you are building a pipeline with all the inputs defined.